circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/primitives/uint/plookup/arithmetic.cpp

```diff
@@ -26,7 +26,7 @@ uint_plookup<Composer, Native> uint_plookup<Composer, Native>::operator+(const u
     const uint256_t overflow = sum >> width;
     const uint256_t remainder = sum & MASK;
 
-    const add_quad gate{
+    const add_quad_<typename Composer::FF> gate{
```

Comment:
For continuity in this class, we should probably define `using FF = typename Composer::FF` and use FF throughout this class